### ISSUE TITLE
Filter Versions Table

### ISSUE THREAD

**Issue by coverbeck** [dockstore/dockstore#2987]
## Feature Request 

The versions table by default shows all tags and branches of a repo. The Broad HCA team would like for the table to show only tags, and on top of that, only specific tags.

### Background

The https://github.com/HumanCellAtlas/skylab repo contains multiple pipelines/workflows. Even though GitHub releases/tags are across the whole repo, they create tags for specific workflows. For example, for [the SmartSeq](https://dockstore.org/workflows/github.com/HumanCellAtlas/skylab/Smartseq2:optimus_1.3.6_terra?tab=versions) workflow in Dockstore, the would only want tags starting with "smartseq" to be displayed. [The optimus workflow](https://dockstore.org/workflows/github.com/HumanCellAtlas/skylab/Optimus:optimus_1.3.6_terra_patch2?tab=info) should only display tags starting with "optimus", etc.

### Solution

Solution is up in the air. Probably some settings in  .dockstore.yml, e.g., something [like this in Travis CI](https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches), although that would only handle not displaying branches, but doesn't filter tags.

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-1059)
┆epic: Versions Table 
┆fixVersions: Dockstore 2.X
┆friendlyId: DOCK-1059
┆sprint: Backlog
┆taskType: Story

**Comment by kbergin** [comment]
Thanks Charles! 

> The versions table by default shows all tags and branches of a workflow. The Broad HCA team would like for the table to show only tags, and on top of that, only specific tags.

Am I correct that the versions table shows all tags and branches of a _repo_ , not just of a workflow? 

Not sure if it's relevant, but I'm representing pipelines for HCA, AoU, GATK, BRAIN initiative, and generally all data produced by the Broad Genomics Platform in this feature request. HCA is the best public example to link to in the background though :)

**Comment by coverbeck** [comment]
> Am I correct that the versions table shows all tags and branches of a _repo_ , not just of a workflow?

Yes, you're correct -- good catch! I  updated the description. Thanks.

**Comment by denis-yuen** [comment]
We've since added an option to only ingest tags, see documentation at 
https://docs.dockstore.org/en/stable/assets/templates/template.html?highlight=tips%20tricks#examples-of-the-filters-field

We do not as yet have a method to only view tags once both branches and tags are ingested, let us know if this works
